Big-Spender/readme.md

```diff
@@ -77,7 +84,11 @@ INSERT YOUR QUERY HERE
 
 ```sql
 INSERT YOUR QUERY HERE
-```
+```SELECT * FROM expense_areas WHERE expense_area = 'Better Hospital Food';
```

Comment:
so this will give us the expense_areas, but how about the transactions? All we will see here is one row with the expense_area, but no transactions amount? 

Hint: you might need to join the spends table ;)